app/components/landing/projectSection/ProjectSection.tsx

```diff
@@ -18,10 +21,39 @@ export type ProjectProps = {
 };
 
 export const ProjectSection = ({ projects }: ProjectProps) => {
+  const [inputValue, setInputValue] = useState('');
+  const [searchResults, setSearchResults] = useState<BasicProject[]>();
```

Comment:
Try creating a project-section-context to store: searchResults, isLoading, refetchProjects, etc

Reply:
do you still think this is necessary? With the recent changes the loading state is only passed to the immediate child component and the search results aren't. Extracting to a dedicated context seems like a lot of boilerplate considering it is not reused. Or should I use the ProjectPageContext instead?